internal/api/services.go

```diff
@@ -25,6 +25,11 @@ type Service struct {
 	Dependencies       []string           `json:"depends" tf:"dependencies"`
 	ActiveAlertSources map[string]string  `json:"-" tf:"active_alert_source_endpoints"`
 	AlertSources       map[string]string  `json:"-" tf:"alert_source_endpoints"`
+	Slack              *SlackChannel      `json:"slack" tf:"-"`
+}
+
+type SlackChannel struct {
+	ChannelID string `json:"channel_id" tf:"-"`
```

Comment:
why tf:"-"`?
doesn't it ignore saving state?

Reply:
Check L62-64, we are storing it there